locales/ja/LC_MESSAGES/user_guide.po

```diff
@@ -1663,15 +1662,14 @@ msgstr "ツール定義を同僚（できれば別の機関の人）に渡し、
 "ックをもらいましょう。"
 
 #: ../../src/topics/best-practices.md:93 f1c1340f9745458980e0fbd641738687
-#, fuzzy
 msgid ""
 "Complex workflows with individual components which can be abstracted should "
 "utilise the [`SubworkflowFeatureRequirement`][subworkflow] to make their "
 "workflow modular and allow sections of them to be easily reused."
 msgstr ""
 "抽象化できる個々のコンポーネントを持つ複雑なワークフローは、ワークフローをモ"
 "ジュール化し、その一部を容易に再利用できるようにするために、[`SubworkflowFeat"
-"ureRequirement`][サブワークフロー]を利用します。"
+"ureRequirement`][subworkflow]を利用します。"
```

Comment:
In `[text][link_target]` the `text` should be translated, but not the `link_target`